### PR TITLE
Try terraforming

### LINKED ISSUE #12: Terraform
to deploy automatically

### PR DESCRIPTION
**Pull request by chaspy**
close: https://github.com/chaspy/delete-branch-bot/issues/12

### CODE REVIEW

**Comment by chaspy** [comment]
hi @kamontia 

I tried to deploy API Gateway resource but something is wrong 😢 
I only have created lambda resource and confirmed that we could create from empty.

please review and merge.
ah, you can ignore the circleci error. this branch has no ci setting.

File: README.md [hunk 1]
```diff
@@ -6,10 +6,18 @@ Automatically delete branch when PR is merged
...
 
-1. Update Code
-2. ./build.sh
-3. Upload handler.zip to AWS Lambda GUI
-4. Save Setting
-5. Test Lambda function with using test.json
+1. Set environment variable for terraformuser
+  - export TF_VAR_aws_access_key="<your access key>"
+  - export TF_VAR_aws_secret_key="<your access secret>"
+1. terrqform init
```
Comment (kamontia) [inline README.md]: terr`q`form is typo

Comment (chaspy) [reply README.md]: thanks! fixed.

**Comment by kamontia** [comment]
@chaspy Thank you for your very big efforts. This implementation gets us to support CI/CD.

**Pull request merged**